spec/features/course/unread_status_management_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Announcements read/unread status management', type: :feature do
+  subject { page }
+
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let!(:first_user) { create(:user, role: :administrator) }
+    let!(:second_user) { create(:user, role: :administrator) }
+    let!(:course) { create(:course) }
+
+    describe 'when visit announcement' do
```

Comment:
context vs. describe. This is a context.

And I think the context string should be 'when visiting announcements'